gitlab/data_source_gitlab_group.go

```diff
@@ -0,0 +1,112 @@
+package gitlab
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/xanzy/go-gitlab"
+	"log"
+)
+
+func dataSourceGitlabGroup() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGitlabGroupRead,
+		Schema: map[string]*schema.Schema{
+			"group_id": {
+				Type:     schema.TypeInt,
+				Computed: true,
+				Optional: true,
+				ConflictsWith: []string{
+					"full_path",
+				},
+			},
+			"full_path": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
+				ConflictsWith: []string{
+					"name",
+				},
+			},
+			"name": {
+				Type:     schema.TypeString,
```

Comment:
should be optional

Reply:
why? it's a computed property only.
same as here: https://github.com/terraform-providers/terraform-provider-gitlab/blob/master/gitlab/data_source_gitlab_user.go#L44

Reply:
I think we could allow searching by name too, don't you think?

Reply:
name is not unique, `group_id` and `full_path` are.

Reply:
Oh yes, you're right!